mscore/musescore.cpp

```diff
@@ -4530,8 +4474,14 @@ int main(int argc, char* av[])
 
       QCoreApplication::setOrganizationName("MuseScore");
       QCoreApplication::setOrganizationDomain("musescore.org");
-      QCoreApplication::setApplicationName("MuseScoreDevelopment");
-      QCoreApplication::setApplicationVersion(VERSION);
+      if (MuseScore::unstable()) {
+            QCoreApplication::setApplicationName("MuseScoreDevelopment");
+            QCoreApplication::setApplicationVersion(QString("Music Score Editor\nUnstable Prerelease for Version %1; Build %2").arg(VERSION).arg(revision));
+            }
+      else {
+            QCoreApplication::setApplicationName("MuseScore2");
+            QCoreApplication::setApplicationVersion(QString("Music Score Editor; Version %1; Build %2").arg(VERSION).arg(revision));
```

Comment:
Mind to explain why this is needed?
 We could do a long version --long-version (and idea for a short switch?) via the old printVersion() function and a short version, -v, --version, via the new QCommandLineParser::addVersionOption(), how does that sound?